seed/seed.js

```diff
@@ -54,18 +54,29 @@ const createTestDataSet = function createTestDataSet() {
   createTestFile('seed/testRecipe.txt', recipeRow);
   createTestFile('seed/testTools.txt', toolRow);
   createTestFile('seed/testJoin.txt', joinRow);
-}
+};
 
 
-// const pipeToDB = function (db, file) {
-//   pg.connect((err, client, done) => {
-//     let stream = client.query(copyFrom('COPY my_table FROM STDIN'));
-//     let fileStream = fs.createReadStream('some_file.tsv');
-//     fileStream.on('error', done);
-//     stream.on('error', done);
-//     stream.on('end', done);
-//     fileStream.pipe(stream);
-//   });
-// };
+const pipeToDB = function pipeToDB(table, file) {
+  const client = new pg.Client({
+    user: 'postgres',
+    host: 'localhost',
+    database: 'megabites',
+  });
+  client.connect((err) => {
+    if (err) {
+      console.log(err);
+    }
+    const stream = client.query(copyFrom(`COPY ${table} FROM STDIN CSV`));
+    const fileStream = fs.createReadStream(file);
+    fileStream.on('error', err => console.error(err));
+    stream.on('error', err => console.error(err));

```

Comment:
I would consider re-ordering the stream event listeners in this area to improve how the code reads